mastodon/src/main/res/values/strings_mo.xml

```diff
@@ -83,6 +84,7 @@
     <string name="mo_setting_default_reply_privacy_summary">Replies will be opted out of discovery features</string>
     <string name="mo_setting_interaction_count_summary">Show how many people interacted with a post in the timeline</string>
     <string name="mo_setting_disable_swipe_summary">Swipe to change viewed timeline</string>
+    <string name="mo_setting_haptic_feedback">Vibrate on boost/favourite/bookmark</string>
```

Comment:
```suggestion
    <string name="mo_setting_haptic_feedback_summary">Vibrate on boost/favourite/bookmark</string>
```
The suffix `summary` helps to indicate this is a settings summary.

Also, possible rename it to `Vibrate on interactiing with a posts`?

Reply:
I changed it to "Vibrate when interacting with a post" - good?